### ISSUE TITLE
[vue/attributes-order]  attrs should sorted separately before/after v-bind

### ISSUE THREAD

**Issue by wsi18n** [vuejs/eslint-plugin-vue#3096]
**What rule do you want to change?**

如果包含 v-bind="$attrs" 时， v-bind="$attrs" 前后的 attributes 应该分开排序
因为 v-bind="xxxx" 的位置会决定了 xxxx 是否覆盖默认属性
When using v-bind="$attrs", attributes before and after v-bind="$attrs" should be sorted separately.
This is because the position of v-bind="xxxx" determines whether xxxx overrides the default attributes.

**Does this change cause the rule to produce more or fewer warnings?**

**How will the change be implemented? (New option, new default behavior, etc.)?**

**Please provide some example code that this change will affect:**

```vue
<div
    a="1"
    v-bind="$attrs"
    v-model="modelValue"
    b="2"
\>
<!-- $attrs can override a but not b, as expected -->
<!-- However, [vue/attributes-order] move the a behind the v-model-->
```

**What does the rule currently do for this code?**

**What will the rule do after it's changed?**

**Additional context**

**Comment by ValentinYoushkevich** [comment]
I can reproduce this with the current version. For the example above, `--fix` gives

```vue
<div
  v-bind="$attrs"
  v-model="modelValue"
  a="1"
  b="2"
/>
```

so `a="1"` moves from before `v-bind="$attrs"` to after it. In Vue 3 whichever binding comes later wins ([migration guide](https://v3-migration.vuejs.org/breaking-changes/v-bind.html)), so the fix changes behavior: `a` used to be overridable by `$attrs` and now it overrides it. `ignoreVBindObject: true` produces the same output, since the other attributes are still sorted across the `v-bind`.

One way to fix it would be to treat `v-bind="object"` as a boundary when `ignoreVBindObject` is enabled and sort attributes only within each side. Would that be acceptable? Happy to send a PR.

**Comment by FloEdelmann** [comment]
Should we maybe just change the fix to a suggestion in this case? Then the user has to contiously decide how to resolve the situation and the code does not get broken automatically.

**Comment by ValentinYoushkevich** [comment]
Sounds good, a suggestion makes sense there. I'll send a PR.
